cves/kernel/CVE-2016-9793.yml

```diff
@@ -106,7 +107,8 @@ vcc_instructions: |
   Place any notes you would like to make in the notes field.
 vccs:
 - commit: 82981930125abfd39d7c8378a9cfdf5e1be2002b
-  note: Discovered automatically by archeogit.
+  note: |
+    Discovered automatically by archeogit.
```

Comment:
Was this manually Confirmed? I would specify

Reply:
It was but I forgot to mention that, thank you!